src/public/modules/forms/base/directives/submit-form.directive.js

```diff
@@ -321,7 +321,10 @@ function submitFormDirective(
             responseMode: form.responseMode,
           },
           submissionContent, // POST body
-        ).then(handleSubmitSuccess, handleSubmitFailure)
+        ).then(
+          (response) => handleSubmitSuccess(response),
+          (err, message) => handleSubmitFailure(err, message),
```

Comment:
this is actually equivalent to the previous version (`.then(handleSubmitSuccess, handleSubmitFailure)`) - let's chat further if you're not sure why!